scripts/tests/ledger_test_apply.sh

```diff
@@ -54,5 +55,5 @@ mkdir $TEMP_FOLDER/genesis/ledger
 tar -zxf  $TEMP_FOLDER/genesis/genesis_ledger_*.tar.gz -C $TEMP_FOLDER/genesis/ledger
 
 echo "running test:"
-time $MINA_APP ledger test apply --ledger-path $TEMP_FOLDER/genesis/ledger  --privkey-path $SENDER --num-txs 200
+time $MINA_APP ledger test apply --ledger-path $TEMP_FOLDER/genesis/ledger  --privkey-path $SENDER --num-txs 20 --dump-benchmark $BENCHMARK_FILE
```

Comment:
```suggestion
time $MINA_APP ledger test apply --ledger-path $TEMP_FOLDER/genesis/ledger  --privkey-path $SENDER --num-txs 200 --dump-benchmark $BENCHMARK_FILE
```

I don't think we should reduce number of txs to 20. If time to run it is a concern, let's make it `125` at least.

Reply:
Yes, another good catch! I decreased for testing purposes